src/openzeppelin/tests/token/test_erc721.cairo

```diff
@@ -714,8 +755,9 @@ fn test_safeTransferFrom_to_receiver_camel() {
 
     assert_state_before_transfer(token_id, owner, receiver);
 
-    set_caller_address(owner);
+    testing::set_caller_address(owner);
     ERC721CamelOnlyImpl::safeTransferFrom(ref state, owner, receiver, token_id, DATA(true));
+    assert_event_transfer(owner, receiver, token_id);
 
     assert_state_after_transfer(token_id, owner, receiver);
```

Comment:
does it make sense to move the assert event transfer into the state after transfer util?
either way, should we normalize parameter order so it's consistent?